src/clr-addons/view-edit-section/view-edit-section.ts

```diff
@@ -1,31 +1,78 @@
 /*
- * Copyright (c) 2018-2019 Porsche Informatik. All Rights Reserved.
+ * Copyright (c) 2018-2022 Porsche Informatik. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
 import { Component, EventEmitter, Input, Output, TemplateRef } from '@angular/core';
+import { animate, state, style, transition, trigger } from '@angular/animations';
 
 @Component({
   selector: 'clr-view-edit-section',
   templateUrl: './view-edit-section.html',
+  animations: [
+    trigger('collapseExpandAnimation', [
+      transition(':enter', [
+        style({ opacity: 0, height: 0, overflow: 'hidden' }),
+        animate('300ms', style({ opacity: 1, height: '*' })),
+      ]),
+      transition(':leave', [
+        style({ opacity: 1, height: '*', overflow: 'hidden' }),
+        animate('300ms', style({ opacity: 0, height: 0 })),
+      ]),
+    ]),
+    trigger('rotateIcon', [
+      state('true', style({ transform: 'rotate(0)' })),
+      state('false', style({ transform: 'rotate(180deg)' })),
+      transition('true => false', animate('300ms ease-out')),
+      transition('false => true', animate('300ms ease-in')),
+    ]),
+  ],
 })
 export class ClrViewEditSection {
   @Input('clrTitle') _title: string;
   @Input('clrSaveText') _saveText = 'Save';
   @Input('clrPreventModeChangeOnSave') _preventSave = false;
   @Input('clrCancelText') _cancelText = 'Cancel';
   @Input('clrEditable') _editable = true;
-  @Input('clrEditMode') _editMode = false;
   @Input('clrEditIcon') _editIcon = 'pencil';
+  @Input('clrIsCollapsible') _isCollapsible = false;
 
   @Input('clrViewRef') viewRef: TemplateRef<any>;
   @Input('clrEditRef') editRef: TemplateRef<any>;
 
+  @Output('clrIsCollapsedChange') _isCollapsedChange = new EventEmitter<boolean>();
   @Output('clrEditModeChange') _editModeChanged = new EventEmitter(false);
   @Output('clrSectionSubmitted') _submitted = new EventEmitter(false);
   @Output('clrSectionEditCancelled') _cancelled = new EventEmitter(false);
 
+  @Input('clrIsCollapsed')
+  public set _isCollapsed(value: boolean) {
```

Comment:
pls refactor to just
@Input('clrIsCollapsed') _isCollapsed = false;

and additionally check the _isCollapsible inside the *ngIf of card-text.
If for any reason "clrIsCollapsed" is set before "clrIsCollapsible" we will never be able to collapse it.